Package fails to install on python 3.11.0a5: fatal error: longintrepr.h: No such file or directory

How to reproduce $subject:

In clean, latest virtual environment:
```
$ python3.11 -m venv ~/.virtualenvs/pyzmq@py311
$ source ~/.virtualenvs/pyzmq@py311/bin/activate
$  pip install --upgrade pip
Requirement already satisfied: pip in ./.virtualenvs/pyzmq@py311/lib/python3.11/site-packages (21.2.3)
Collecting pip
  Downloading pip-22.0.3-py3-none-any.whl (2.1 MB)
     |████████████████████████████████| 2.1 MB 1.3 MB/s 
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 21.2.3
    Uninstalling pip-21.2.3:
      Successfully uninstalled pip-21.2.3
Successfully installed pip-22.0.3
$ python --version
Python 3.11.0a5
$ cat /etc/fedora-release 
Fedora release 35 (Thirty Five)
```

install `pymzq`:

```
$ pip install pyzmq
Collecting pyzmq
  Downloading pyzmq-22.3.0.tar.gz (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 2.1 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: pyzmq
  Building wheel for pyzmq (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for pyzmq (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [192 lines of output]
      /tmp/pip-build-env-26zcrx59/overlay/lib/python3.11/site-packages/setuptools/_distutils/dist.py:275: UserWarning: Unknown distribution option: 'cffi_modules'
        warnings.warn(msg)
      running bdist_wheel
      running build
      running build_py
      creating build/lib.linux-x86_64-3.11
      creating build/lib.linux-x86_64-3.11/zmq
      copying zmq/error.py -> build/lib.linux-x86_64-3.11/zmq
      copying zmq/decorators.py -> build/lib.linux-x86_64-3.11/zmq
      copying zmq/asyncio.py -> build/lib.linux-x86_64-3.11/zmq
      copying zmq/_future.py -> build/lib.linux-x86_64-3.11/zmq
      copying zmq/__init__.py -> build/lib.linux-x86_64-3.11/zmq
      creating build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/z85.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/win32.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/strtypes.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/monitor.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/jsonapi.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/interop.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/garbage.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/constant_names.py -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/__init__.py -> build/lib.linux-x86_64-3.11/zmq/utils
      creating build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_zmqstream.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_z85.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_win32_shim.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_version.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_ssh.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_socket.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_security.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_retry_eintr.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_reqrep.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_pubsub.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_proxy_steerable.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_poll.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_pair.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_mypy.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_multipart.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_monqueue.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_monitor.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_message.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_log.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_ioloop.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_includes.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_imports.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_future.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_etc.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_error.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_draft.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_device.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_decorators.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_cython.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_context.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_constants.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_cffi_backend.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_auth.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/test_asyncio.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/conftest.py -> build/lib.linux-x86_64-3.11/zmq/tests
      copying zmq/tests/__init__.py -> build/lib.linux-x86_64-3.11/zmq/tests
      creating build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/version.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/tracker.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/stopwatch.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/socket.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/poll.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/frame.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/context.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/constants.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/attrsettr.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/__init__.py -> build/lib.linux-x86_64-3.11/zmq/sugar
      creating build/lib.linux-x86_64-3.11/zmq/ssh
      copying zmq/ssh/tunnel.py -> build/lib.linux-x86_64-3.11/zmq/ssh
      copying zmq/ssh/forward.py -> build/lib.linux-x86_64-3.11/zmq/ssh
      copying zmq/ssh/__init__.py -> build/lib.linux-x86_64-3.11/zmq/ssh
      creating build/lib.linux-x86_64-3.11/zmq/log
      copying zmq/log/handlers.py -> build/lib.linux-x86_64-3.11/zmq/log
      copying zmq/log/__main__.py -> build/lib.linux-x86_64-3.11/zmq/log
      copying zmq/log/__init__.py -> build/lib.linux-x86_64-3.11/zmq/log
      creating build/lib.linux-x86_64-3.11/zmq/green
      copying zmq/green/poll.py -> build/lib.linux-x86_64-3.11/zmq/green
      copying zmq/green/device.py -> build/lib.linux-x86_64-3.11/zmq/green
      copying zmq/green/core.py -> build/lib.linux-x86_64-3.11/zmq/green
      copying zmq/green/__init__.py -> build/lib.linux-x86_64-3.11/zmq/green
      creating build/lib.linux-x86_64-3.11/zmq/green/eventloop
      copying zmq/green/eventloop/zmqstream.py -> build/lib.linux-x86_64-3.11/zmq/green/eventloop
      copying zmq/green/eventloop/ioloop.py -> build/lib.linux-x86_64-3.11/zmq/green/eventloop
      copying zmq/green/eventloop/__init__.py -> build/lib.linux-x86_64-3.11/zmq/green/eventloop
      creating build/lib.linux-x86_64-3.11/zmq/eventloop
      copying zmq/eventloop/zmqstream.py -> build/lib.linux-x86_64-3.11/zmq/eventloop
      copying zmq/eventloop/ioloop.py -> build/lib.linux-x86_64-3.11/zmq/eventloop
      copying zmq/eventloop/future.py -> build/lib.linux-x86_64-3.11/zmq/eventloop
      copying zmq/eventloop/_deprecated.py -> build/lib.linux-x86_64-3.11/zmq/eventloop
      copying zmq/eventloop/__init__.py -> build/lib.linux-x86_64-3.11/zmq/eventloop
      creating build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado
      copying zmq/eventloop/minitornado/util.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado
      copying zmq/eventloop/minitornado/stack_context.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado
      copying zmq/eventloop/minitornado/log.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado
      copying zmq/eventloop/minitornado/ioloop.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado
      copying zmq/eventloop/minitornado/concurrent.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado
      copying zmq/eventloop/minitornado/__init__.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado
      creating build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado/platform
      copying zmq/eventloop/minitornado/platform/windows.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado/platform
      copying zmq/eventloop/minitornado/platform/posix.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado/platform
      copying zmq/eventloop/minitornado/platform/interface.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado/platform
      copying zmq/eventloop/minitornado/platform/common.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado/platform
      copying zmq/eventloop/minitornado/platform/auto.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado/platform
      copying zmq/eventloop/minitornado/platform/__init__.py -> build/lib.linux-x86_64-3.11/zmq/eventloop/minitornado/platform
      creating build/lib.linux-x86_64-3.11/zmq/devices
      copying zmq/devices/proxysteerabledevice.py -> build/lib.linux-x86_64-3.11/zmq/devices
      copying zmq/devices/proxydevice.py -> build/lib.linux-x86_64-3.11/zmq/devices
      copying zmq/devices/monitoredqueuedevice.py -> build/lib.linux-x86_64-3.11/zmq/devices
      copying zmq/devices/monitoredqueue.py -> build/lib.linux-x86_64-3.11/zmq/devices
      copying zmq/devices/basedevice.py -> build/lib.linux-x86_64-3.11/zmq/devices
      copying zmq/devices/__init__.py -> build/lib.linux-x86_64-3.11/zmq/devices
      creating build/lib.linux-x86_64-3.11/zmq/backend
      copying zmq/backend/select.py -> build/lib.linux-x86_64-3.11/zmq/backend
      copying zmq/backend/__init__.py -> build/lib.linux-x86_64-3.11/zmq/backend
      creating build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/__init__.py -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      creating build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/utils.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/socket.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/message.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/error.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/devices.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/context.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/constants.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/_poll.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      copying zmq/backend/cffi/__init__.py -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      creating build/lib.linux-x86_64-3.11/zmq/auth
      copying zmq/auth/thread.py -> build/lib.linux-x86_64-3.11/zmq/auth
      copying zmq/auth/ioloop.py -> build/lib.linux-x86_64-3.11/zmq/auth
      copying zmq/auth/certs.py -> build/lib.linux-x86_64-3.11/zmq/auth
      copying zmq/auth/base.py -> build/lib.linux-x86_64-3.11/zmq/auth
      copying zmq/auth/asyncio.py -> build/lib.linux-x86_64-3.11/zmq/auth
      copying zmq/auth/__init__.py -> build/lib.linux-x86_64-3.11/zmq/auth
      copying zmq/__init__.pxd -> build/lib.linux-x86_64-3.11/zmq
      copying zmq/__init__.pyi -> build/lib.linux-x86_64-3.11/zmq
      copying zmq/py.typed -> build/lib.linux-x86_64-3.11/zmq
      copying zmq/utils/buffers.pxd -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/zmq_constants.h -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/zmq_compat.h -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/pyversion_compat.h -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/mutex.h -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/ipcmaxlen.h -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/getpid_compat.h -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/config.json -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/utils/compiler.json -> build/lib.linux-x86_64-3.11/zmq/utils
      copying zmq/sugar/constants.pyi -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/sugar/__init__.pyi -> build/lib.linux-x86_64-3.11/zmq/sugar
      copying zmq/devices/monitoredqueue.pxd -> build/lib.linux-x86_64-3.11/zmq/devices
      copying zmq/backend/constants.pyi -> build/lib.linux-x86_64-3.11/zmq/backend
      copying zmq/backend/__init__.pyi -> build/lib.linux-x86_64-3.11/zmq/backend
      copying zmq/backend/cython/socket.pxd -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/message.pxd -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/libzmq.pxd -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/context.pxd -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/checkrc.pxd -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/__init__.pxd -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/constants.pxi -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cython/constant_enums.pxi -> build/lib.linux-x86_64-3.11/zmq/backend/cython
      copying zmq/backend/cffi/_cdefs.h -> build/lib.linux-x86_64-3.11/zmq/backend/cffi
      running build_ext
      running configure
      Settings obtained from pkg-config: {'library_dirs': [], 'include_dirs': [], 'libraries': ['zmq']}
      {'libraries': ['zmq'], 'include_dirs': [], 'library_dirs': [], 'runtime_library_dirs': [], 'extra_link_args': []}
      Configure: Autodetecting ZMQ settings...
          Custom ZMQ dir:
      Checking for timer_create
      ** Errors about missing timer_create are a normal part of this process **
      ************************************************
      creating build/temp.linux-x86_64-3.11/scratch/tmp
      gcc -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -c /tmp/timer_createypnicvk3.c -o build/temp.linux-x86_64-3.11/scratch/tmp/timer_createypnicvk3.o
      /tmp/timer_createypnicvk3.c: In function ‘main’:
      /tmp/timer_createypnicvk3.c:2:5: warning: implicit declaration of function ‘timer_create’ [-Wimplicit-function-declaration]
          2 |     timer_create();
            |     ^~~~~~~~~~~~
      gcc build/temp.linux-x86_64-3.11/scratch/tmp/timer_createypnicvk3.o -o build/temp.linux-x86_64-3.11/scratch/a.out
      gcc -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -Izmq/utils -c build/temp.linux-x86_64-3.11/scratch/vers.c -o build/temp.linux-x86_64-3.11/scratch/vers.o
      gcc build/temp.linux-x86_64-3.11/scratch/vers.o -lzmq -o build/temp.linux-x86_64-3.11/scratch/vers
          ZMQ version detected: 4.3.4
      ************************************************
      building 'zmq.backend.cython._device' extension
      creating build/temp.linux-x86_64-3.11/zmq
      creating build/temp.linux-x86_64-3.11/zmq/backend
      creating build/temp.linux-x86_64-3.11/zmq/backend/cython
      gcc -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -DHAVE_SYS_UN_H=1 -Izmq/utils -I/home/pbabinca/.virtualenvs/pyzmq@py311/include -I/usr/include/python3.11 -c zmq/backend/cython/_device.c -o build/temp.linux-x86_64-3.11/zmq/backend/cython/_device.o
      zmq/backend/cython/_device.c:217:12: fatal error: longintrepr.h: No such file or directory
        217 |   #include "longintrepr.h"
            |            ^~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/lib64/ccache/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyzmq
Failed to build pyzmq
ERROR: Could not build wheels for pyzmq, which is required to install pyproject.toml-based projects
```

While I don't understand the issue, similar bug against Fedora was rejected according to (their rpm) packaging guidelines https://bugzilla.redhat.com/show_bug.cgi?id=2028069 :
```
This is from pre-built Cython sources; these should be removed in %prep with e.g.

    rm $(grep -rl '/\* Generated by Cython')

These files will then be rebuilt from source, with a updated version of Cython.
See: https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_using_cython
```

(Yes, I understand, that I'm installing this from PyPI)

Header file is also referenced in 7th bullet point of [Porting to Python 3.11](https://docs.python.org/3.11/whatsnew/3.11.html#id2).

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -14,8 +14,8 @@ on:
       - zmq/utils/*.h
 
 env:
-  cython: "0.29.24"
-  cibuildwheel: "2.2.2"
+  cython: "0.29.28"
+  cibuildwheel: "2.3.1"
   TWINE_NONINTERACTIVE: "1"
 
 jobs:

